src/hooks/useUserByAddress.ts

```diff
@@ -0,0 +1,21 @@
+import { useGetUserByAddressQuery } from '~gql';
+import { Address } from '~types';
+
+const useUserByAddress = (address: Address) => {
```

Comment:
We have discussed previously the pointlessness of wrapping queries in hooks. This is one of those cases, no?

Reply:
Yeah, that's right. I guess I just followed the existing pattern of this being a hook. However, I think it makes sense for the address query to stay wrapped in a hook, since it's being used in multiple places (which would otherwise lead to repetition of the boilerplate)